.github/workflows/techlabblog-deploy-dev.yml

```diff
@@ -0,0 +1,57 @@
+name: Techlab Blog | Deploy Dev
+
+on:
+  push:
+    branches: [ft/techlabblog-workflow]
+    paths:
+      - "apps/techlabblog/**"
+      - "Dockerfile"
+      - ".github/workflows/techlabblog-deploy-dev.yml"
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: true
+
+env:
+  SENTRY_ENVIRONMENT: "development"
+
+jobs:
+  env-setup:
+    runs-on: ubuntu-latest
+    outputs:
+      DOKKU_REMOTE_URL: ${{ steps.env-setup.outputs.DOKKU_REMOTE_URL }}
+      DOKKU_PRIVATE_KEY: ${{ steps.env-setup.outputs.DOKKU_PRIVATE_KEY }}
+      SENTRY_ORG: ${{ steps.env-setup.outputs.SENTRY_ORG }}
+      SENTRY_DSN: ${{ steps.env-setup.outputs.SENTRY_DSN }}
+      SENTRY_AUTH_TOKEN: ${{ steps.env-setup.outputs.SENTRY_AUTH_TOKEN }}
+      SENTRY_ENVIRONMENT: ${{ steps.env-setup.outputs.SENTRY_ENVIRONMENT }}
+    steps:
+      - name: Environment Setup
+        id: env-setup
+        run: |
+          echo "SENTRY_ORG=${{ secrets.SENTRY_ORG }}" >> $GITHUB_ENV
+          echo "SENTRY_DSN=${{ secrets.TECHLABBLOG_SENTRY_DSN }}" >> $GITHUB_ENV
+          echo "SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}" >> $GITHUB_ENV
+          echo "SENTRY_ENVIRONMENT=${{ env.SENTRY_ENVIRONMENT }}" >> $GITHUB_ENV
```

Comment:
1. I think only `SENTRY_AUTH_TOKEN` is a secret; the rest should just be envs
2. Since it's org-wide secret, we can use inherit to pass it to called workflows.
>Workflows that call reusable workflows in the **same** organization or enterprise can use the `inherit` keyword to implicitly pass the secrets.

Reply:
I think the `SENTRY_DSN` should also be a secret

Reply:
Not according to [Sentry](https://docs.sentry.io/concepts/key-terms/dsn-explainer/#dsn-utilization).